sources/preprocessing/sequence.md

```diff
@@ -57,10 +56,10 @@ Vector Space](http://arxiv.org/pdf/1301.3781v3.pdf)
 keras.preprocessing.sequence.make_sampling_table(size, sampling_factor=1e-5)
 ```
 
-Used for generating the `sampling_table` argument for `skipgrams`. `sampling_table[i]` is the probability of sampling the word i-th most common word in a dataset (more common words should be sampled less frequently, for balance).
+`skipgrams`の`sampling_table`引数を生成するために利用します．`sampling_table[i]`はデータセット中でi番目に頻出する単語をサンプリングする確率です（バランスを保つために，より頻出する語はこれより低い頻度でサンプリングされます）．
 
-- __Return__: numpy array of shape `(size,)`.
+- __戻り値__: 形状が`(size,)`であるnumpy配列．
 
-- __Arguments__:
-    - __size__: size of the vocabulary considered.
-    - __sampling_factor__: lower values result in a longer probability decay (common words will be sampled less frequently). If set to 1, no subsampling will be performed (all sampling probabilities will be 1).
+- __引数__:
+    - __size__: 語彙数．
+    - __sampling_factor__: lower values result in a longer probability decay (common words will be sampled less frequently). 1が与えられたとき，サブサンプリングは行われません（全てのサンプリング確率が1になります）．
```

Comment:
It means: with lower values of `sampling_factor`, common (frequent) words will be sampled less often. Is that clear?